pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdTopics.java

```diff
@@ -1097,10 +1099,23 @@ private class PeekMessages extends CliCommand {
         @Option(names = { "-n", "--count" }, description = "Number of messages (default 1)", required = false)
         private int numMessages = 1;
 
+        @Option(names = { "-ssm", "--show-server-marker" },
+                description = "Enables the display of internal server write markers.", required = false)
+        private boolean showServerMarker = false;
+
+        @Option(names = { "-sta", "--show-txn-aborted" },
+                description = "Enables the display of messages from aborted transactions.", required = false)
+        private boolean showTxnAborted = false;
+
+        @Option(names = { "-stu", "--show-txn-uncommitted" },
+                description = "Enables the display of messages from uncommitted transactions.", required = false)
+        private boolean showTxnUncommitted = false;
```

Comment:
I would like suggest to use `--transaction-isolation-level` and the default value will be `ReadCommitted`, if users want peek the uncommitted messages, they can change to `--transaction-isolation-level=ReadUnCommitted`

Reply:
Sorry for the late review of the proposal.

Reply:
Although the name "--transaction-isolation-level" is quite standard, it seems to be inconsistent with the other two parameters.

Are there more considerations? For example, in the future, transactions might involve more isolation features.

Reply:
> Are there more considerations? For example, in the future, transactions might involve more isolation features.

Yes, the major consideration is the consistency in the future. There are some discussions under the mailing list, e.g. https://lists.apache.org/thread/8ny0qtp7m9qcdbvnfjdvpnkc4c5ssyld. We'd better avoid using different terms for transaction messages.

> Although the name "--transaction-isolation-level" is quite standard, but it seems to be inconsistent with the other two parameters.

We can replace `--show-txn-aborted` and `--show-txn-uncommitted` with "--transaction-isolation-level".
For `--show-server-marker`, I think it should be fine.

Reply:
Ok, I will apply it to this PR(including PIP change.)